instantmusic-0.1/bin/instantmusic

```diff
@@ -178,13 +178,11 @@ def query_and_download(search, has_prompts=True, is_quiet=False):
             except:
                 print ('Cant get album name')
             try:
+                test1 = ('')
```

Comment:
Can we have more verbose variable names ? I know it wasn't much verbose before either, but now is a good opportunity to fix that.